pkg/network/node/egressip.go

```diff
@@ -151,9 +151,13 @@ func (eip *egressIPWatcher) SetNamespaceEgressDropped(vnid uint32) {
 	}
 }
 
-func (eip *egressIPWatcher) SetNamespaceEgressViaEgressIP(vnid uint32, egressIP, nodeIP string) {
-	mark := eip.iptablesMark[egressIP]
-	if err := eip.oc.SetNamespaceEgressViaEgressIP(vnid, nodeIP, mark); err != nil {
+func (eip *egressIPWatcher) SetNamespaceEgressViaEgressIP(vnid uint32, activeEgressIPs map[string]string) {
+	marks := make(map[string]string)
```

Comment:
(same comment about map-vs-array-of-struct)

Reply:
done